packages/opentelemetry-semantic-conventions/src/trace/faas.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Database attribute names defined by the Opetelemetry Semantic Conventions specification
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/faas.md
+ */
+export const FaasAttribute = {
+  // General attributes
+
+  /**
+   * The name of the function being executed.
+   *
+   * @remarks
+   * Required.
+   */
+  FAAS_NAME: 'faas.name',
+
+  /**
+   * The unique ID of the function being executed.
+   * For example, in AWS Lambda this field corresponds to the ARN value, in GCP to the URI of the resource, and in Azure to the FunctionDirectory field.
+   *
+   * @remarks
+   * Required.
+   */
+  FAAS_ID: 'faas.id',
+
+  /**
+   * The version string of the function being executed.
+   *
+   * @remarks
+   * Optional.
+   */
+  FAAS_VERSION: 'faas.version',
+
+  /**
+   * The execution environment ID as a string.
+   *
+   * @remarks
+   * Optional.
+   */
+  FAAS_INSTANCE: 'faas.instance',
+
+  /**
+   * Type of the trigger on which the function is executed.
+   * Possible values: datasource, http, pubsub, timer, other
+   *
+   * @remarks
+   * Required if applicable.
+   */
+  FAAS_TRIGGER: 'faas.trigger',
+
+  /**
+   * The execution ID of the current function execution.
+   *
+   * @remarks
+   * Optional.
+   */
+  FAAS_EXECUTION: 'faas.execution',
+
+  // Incoming Invocations
+
+  /**
+   * A boolean that is true if the serverless function is executed for the first time (aka cold-start).
+   *
+   * @remarks
+   * Optional.
+   */
+  FAAS_COLD_START: 'faas.coldstart',
+
+  // Outgoing Invocations
+  // This section describes outgoing FaaS invocations as they are reported by a client calling a FaaS instance.
+
+  /**
+   * The name of the invoked function.
+   * SHOULD be equal to the faas.name resource attribute of the invoked function.
+   *
+   * @remarks
+   * Optional.
+   */
+  FASS_INVOKED_NAME: 'faas.invoked_name',
```

Comment:
```suggestion
  FASS_INVOKED_NAME: 'faas.invoked_name',
```

FASS should be FAAS I think? same for two others below

Reply:
thanks. fixed @naseemkullah